dune

```diff
@@ -8,6 +8,7 @@
     (chdir
      vendor/cvc5
      (progn
+      (run pip install -r contrib/requirements_build.txt)
```

Comment:
Arch doesn't seem to like this

Reply:
We can have OS dependent rules like:
```
(rule
  (enabled_if (= %{ocaml-config:system} macosx))
```
But I think this would mean duplicating the whole rule

Reply:
Yeah, it would be a possibility. However, in archlinux, I also get the missing python library error, but running pip in this system-wide manner doesn't work. So, I need to find a way to address this issue for every OS build target we support, ensuring it works  across all OSs